recipe/meta.yaml

```diff
@@ -63,9 +62,6 @@ test:
   commands:
     # skipping pip check because of error
     # pdfplumber X.X.X has requirement pdfminer.six==YYYYMMDD, but you have pdfminer-six 0.0.1.
```

Comment:
we should have checked this, and fix `pdfminer-six`

Reply:
@Cansisti We could try to uncomment this one according to @ifitchet 's comment on previous PR: https://github.com/AnacondaRecipes/layoutparser-feedstock/pull/1/files#r1928429947

Reply:
Done